xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.cpp

```diff
@@ -151,7 +151,7 @@ bool CDVDInputStreamNavigator::Open()
   {
     // if dvd image file (ISO or alike) open using libdvdnav stream callback functions
     m_pstream = std::make_unique<CDVDInputStreamFile>(
-        m_item, XFILE::READ_TRUNCATED | XFILE::READ_BITRATE | XFILE::READ_CHUNKED);
+        m_item, XFILE::READ_TRUNCATED | XFILE::READ_BITRATE | XFILE::READ_NO_CACHE);
```

Comment:
Why is this changed to no-cache?

Reply:
If I'm not mistaken FileCache should not be used for ISO images because it produces long delays when trying to access to non-contiguous areas (small files inside the main ISO container). Then seems the original intention of `READ_CHUNKED` flag here is disable caching probably because documentation says this:

https://github.com/xbmc/xbmc/blob/adad147d4f521c3d24ce5d2325efc9e9a92f7468/xbmc/filesystem/IFileTypes.h#L19-L20

Although currently FileCache is not disabled thanks to this but rather due to not setting READ_AUDIO_VIDEO flag.

Then `READ_NO_CACHE` is not mandatory here but it doesn't seem to harm.

Reply:
Perhaps this should be in a separate commit then explaining this.

Reply:
OK. Removed READ_NO_CACHE flag for now. This only affects DVD ISOs, Blu-Ray ISOs are handled by libbluray instead and is not using FileCache anyway.